core/src/test/java/org/everit/json/schema/loader/SchemaLoaderTest.java

```diff
@@ -687,14 +655,18 @@ public void unknownMetaSchemaException() {
     }
 
     @Test
-    public void syntheticAllOf() {
```

Comment:
@snazarenko67 can you please restore this test?

Reply:
Hi!
I will look at all the comments and answer ASAP.